features/networking/service.feature

```diff
@@ -153,3 +153,433 @@ Feature: Service related networking scenarios
     Then the step should succeed
     And the output should contain:
       | 8080 |
+
+  # @author weliang@redhat.com
+  # @case_id OCP-24668
+  @admin
+  Scenario: externalIP defined in service but no sepc.externalIP defined	
+    Given I switch to the first user
+    Given I have a project 
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    
+    # Create a service with a externalIP in :proj1
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json | 
+      | n | <%= cb.proj1 %> |
+    Then the output should contain "externalIPs have been disabled" 
+    
+    
+  # @author weliang@redhat.com
+  # @case_id OCP-24669
+  @admin
+  Scenario: externalIP defined in service with set ExternalIP in allowedCIDRs	 
+    # Create additional network through CNO
+    When I run the :patch admin command with:
+      | resource      | networks.config.openshift.io |
+      | resource_name | cluster           |
+      | p             | {"spec":{"externalIP":{"policy":{"allowedCIDRs":["22.2.2.0/24"]}}}} |
+      | type          | merge | 
+    Then I wait for the steps to pass:
+    """
+    When I run the :get admin command with:
+      | resource      | networks.config.openshift.io                |
+      | resource_name | cluster             |
+      | template      | {{{{.spec.externalIP.policy.allowedCIDRs}}}} |
+      | o             | yaml   |
+    And the output should contain "22.2.2.0"
+    """
+ 
+    # Need wait for about 300 seconds to apply new additional network configuration
+    Given 300 seconds have passed
+    
+    # Create a svc with externalIP/22.2.2.10
+    Given I switch to the first user
+    Given I have a project 
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    When I download a file from "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/externalip_service1.json"
+    And I replace lines in "externalip_service1.json":
+      | 10.5.0.1 | 22.2.2.10 |
+    And I run the :create client command with:
+      | f | externalip_service1.json |
+      | n | <%= cb.proj1 %> |
+    Then I wait for the steps to pass:
```

Comment:
if the externalip cannot be added , then it will be hang there. Can we set timeout here

Reply:
Update to "Then I wait up 10 seconds for the steps to pass:"